android/gradle.properties

```diff
@@ -1,4 +1,5 @@
 android.useDeprecatedNdk=true
+android.enableAapt2=false
```

Comment:
What did you need to add this? Looks like this flag will be removed at the end of the year so AAPT2 will always be enabled 🤔

Reply:
Yeah, @sjchmiela already told me not to do so.. Well, it's all because of the problem that i described here: https://github.com/expo/expo/issues/2956